### DISCUSSION TITLE
Build for Windows target on Arch Linux

### DISCUSSION THREAD

**Discussion by farzadmf** [rusqlite/rusqlite#1100] [Questions] [answered]
Hi, sorry if this is a stupid question. I'm not very familiar with `rust`, and I tried to familiarize myself by trying to help on a repo ([https://github.com/SamHH/bukubrow-host](https://github.com/SamHH/bukubrow-host))

I'm trying to add Windows support, and I'm trying to do that on Arch Linux. What I did was to first add the target
```
rustup target add x86_64-pc-windows-msvc
````
but when I try to build, I see an error. More detailed information is available on the issue I opened there: SamHH/bukubrow-host#5.

That repo is using this library with `features = ["bundled"]`, and when building, from what I understand, it's trying to locate a file:
```
.../target/x86_64-pc-windows-msvc/release/build/libsqlite3-sys-26abfe4612b4a8af/out/sqlite3/sqlite3.o
```

but it cannot find it. Again, maybe this is a trivial question and only because of my lack of knowledge about `rust`, but I'd appreciate your help.

Thank you

**Comment by gwenn** [comment, marked as answer]
https://wiki.archlinux.org/index.php/Rust#Cross_compiling

**Comment by farzadmf** [comment]
Thank you @gwenn for your reply, I will check that out. Is there also a resource for Ubuntu?
